model/RestaurantManagerModel.ts

```diff
@@ -27,7 +27,7 @@ class RestaurantManagerModel {
                 restaurantOwnerID: String,
                 restuarantID: String,
             },
-            { collection: "RestaurantManagers" }
+            { collection: "RestaurantManager" }
```

Comment:
For some of the other model files, the collection name is plural. Just wanted to check, is this singular for any specific reason?